recipe/meta.yaml

```diff
@@ -9,37 +9,44 @@ package:
 source:
   fn: {{ name }}-{{ version }}.{{ bundle }}
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ bundle }}
-  sha256: 6d33aef15b5bcead780acc339464aae8a6e28f13c90d8b1cf9de8b549d1c0b4b
+  sha256: c0a7baa9fdc448cdd3dd6f0939df72eec5177b2f7abe6cb82fc934d29caac9c3
 
 build:
   number: 0
   script: "{{ PYTHON }} -m pip install . -vv"
-  noarch: python
+  skip: true  # [py<37]
 
 requirements:
   host:
     - python
     - pip
-    - flask
+    - flask >=1.0.4
+    - wheel
 
   run:
     - python
     - flask
```

Comment:
`flask >=1.0.4` and `werkzeug >=1.0.1` are required, see https://github.com/maxcountryman/flask-login/blob/91541785cec4153cc148485929ec44c1c734b410/setup.py#L7